pkg/gce-pd-csi-driver/node.go

```diff
@@ -340,6 +340,14 @@ func (ns *GCENodeServer) NodeStageVolume(ctx context.Context, req *csi.NodeStage
 				devicePath, stagingTargetPath, fstype, options, err))
 	}
 
+	// Part 4: Resize filesystem.
```

Comment:
I'd add a reason why we have this call here in addition to the NodeExpandVolume resize call, I think we can trace it back through blame too though

Reply:
I've added a link to the original issue in k/k where the reasons are explained.